engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -313,15 +314,19 @@ public static void main(String[] args) throws Throwable
         stramChild.teardown();
       }
     } catch (Error error) {
+      //fetch logFileInfo before logging exception, to get offset before exception
+      LogFileInformation logFileInfo = LoggerUtil.getLogFileInformation();
```

Comment:
if error level is not enabled, the object we receive will have "null" file name. So I guess no need to check for logger level.